ims/einstein/operations.py

```diff
@@ -216,6 +214,8 @@ def shutdown(self):
     @log
     def provision(self, node_name, img_name, network, nic):
         try:
+            mac_addr = ("01-" + self.hil.get_node_mac_addr(node_name).
+                        replace(":", "-"))
```

Comment:
I added that to split it for pep8. Otherwise I don't know how to split that line. want me to use `\`?